src/test/java/uk/gov/crowncommercial/dts/scale/cat/controller/ProjectsControllerTest.java

```diff
@@ -192,15 +190,23 @@ void updateProcurementProjectName_200_OK() throws Exception {
   @Test
   void listProcurementEventTypes_200_OK() throws Exception {
 
+    var expectedJson = "[" + "{\"type\":\"EOI\",\"description\":\"Expression of Interest\","
+        + "\"preMarketActivity\":true},"
+        + "{\"type\":\"RFI\",\"description\":\"Request for Information\","
+        + "\"preMarketActivity\":true},"
+        + "{\"type\":\"CA\",\"description\":\"Capability Assessment\",\"preMarketActivity\":true},"
+        + "{\"type\":\"DA\",\"description\":\"Direct Award\",\"preMarketActivity\":true},"
+        + "{\"type\":\"FC\",\"description\":\"Further Competition\",\"preMarketActivity\":true}]";
+
     when(procurementProjectService.getProjectEventTypes(PROC_PROJECT_ID))
-        .thenReturn(Arrays.asList(ViewEventType.values()));
+        .thenReturn(TestUtils.getEventTypes());
 
     mockMvc
         .perform(get("/tenders/projects/" + PROC_PROJECT_ID + "/event-types")
             .with(validJwtReqPostProcessor).accept(APPLICATION_JSON))
         .andDo(print()).andExpect(status().isOk())
-        .andExpect(content().contentType(APPLICATION_JSON)).andExpect(jsonPath("$.size()").value(6))
-        .andExpect(jsonPath("$[*]", contains("EOI", "RFI", "CA", "DA", "FC", "TBD")));
+        .andExpect(content().contentType(APPLICATION_JSON)).andExpect(jsonPath("$.size()").value(5))
+        .andExpect(content().string(expectedJson));
```

Comment:
Ok with this for now, but the direct string comparison makes the test somewhat brittle - e.g. if the order of fields in `EventType` changed, this could break. There is a technique that uses the JSON path expressions with the `&&` operator to test for the presence of complex objects in an array - something like https://stackoverflow.com/questions/29703891/jsonpath-and-operator-on-array